vtex-apps/store
# Trigger productView when selected SKU changes

#### What is the purpose of this pull request?

Track when user selects another SKU in the product page.

#### What problem is this solving?

Previously we just tracked the product of the product page. This change makes it possible to track all the navigation between SKUs.

#### How should this be manually tested?

1. Open the workspace [storecomponents/breno](https://breno--storecomponents.myvtex.com/classic-shoes/p)
2. Open the Dev Tools console and check the variable `dataLayer`
3. Check for the event: 
![image](https://user-images.githubusercontent.com/284515/64535981-aa234b80-d2ee-11e9-8405-f1a932fa4497.png)
4. Change the selected SKU by clicking in another shoes color
5. Check the `dataLayer` again.
6. Verify that another `productDetail` event was triggered
![image](https://user-images.githubusercontent.com/284515/64536065-cde69180-d2ee-11e9-9633-7b4eef220ea8.png)

#### Types of changes
